### ISSUE TITLE
NPE when clicking any button on first screen

### ISSUE THREAD

**Issue by rohieb** [00-Evan/shattered-pixel-dungeon-gdx#17]
After starting the jar file, or running the gradle wrapper, the main windows shows the screen about what is new in this version, along with two buttons, "Continue" and "Changelist". When I click either of those buttons, the application crashes with a NullPointerException:

```
$ ./gradlew desktop:run
Picked up _JAVA_OPTIONS:  -Dawt.useSystemAAFontSettings=on -Djava.net.preferIPv4Addresses=true
Configuration on demand is an incubating feature.
:PD-classes:compileJava UP-TO-DATE
:PD-classes:processResources UP-TO-DATE
:PD-classes:classes UP-TO-DATE
:PD-classes:jar UP-TO-DATE
:core:compileJava UP-TO-DATE
:core:processResources UP-TO-DATE
:core:classes UP-TO-DATE
:core:jar UP-TO-DATE
:desktop:compileJava UP-TO-DATE
:desktop:processResources UP-TO-DATE
:desktop:classes UP-TO-DATE
:desktop:run
Picked up _JAVA_OPTIONS:  -Dawt.useSystemAAFontSettings=on -Djava.net.preferIPv4Addresses=true
Exception in thread "LWJGL Application" java.lang.NullPointerException
	at com.shatteredpixel.shatteredpixeldungeon.scenes.WelcomeScene.updateVersion(WelcomeScene.java:159)
	at com.shatteredpixel.shatteredpixeldungeon.scenes.WelcomeScene.access$000(WelcomeScene.java:39)
	at com.shatteredpixel.shatteredpixeldungeon.scenes.WelcomeScene$3.onClick(WelcomeScene.java:105)
	at com.watabou.noosa.ui.Button$1.onClick(Button.java:62)
	at com.watabou.noosa.TouchArea.onSignal(TouchArea.java:130)
	at com.watabou.noosa.TouchArea.onSignal(TouchArea.java:27)
	at com.watabou.utils.Signal.dispatch(Signal.java:80)
	at com.watabou.pd.desktop.DesktopInputProcessor.touchUp(DesktopInputProcessor.java:143)
	at com.badlogic.gdx.backends.lwjgl.LwjglInput.processEvents(LwjglInput.java:332)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:215)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:124)

BUILD SUCCESSFUL

Total time: 10.036 secs
```

I can remember that it didn't do this with the downloaded 0.3.x jars as well as the 0.4.1 jar until recently (I already played 0.4.1 before on this machine), so I figure it must be related to some update on my system, but I'm not sure how to debug this further. I tried deleting `~/.watabou/` as well as `/tmp/libgdx*` (in case this is related), but that didn't solve the issue.

Further information:
```
$ java -version
Picked up _JAVA_OPTIONS:  -Dawt.useSystemAAFontSettings=on -Djava.net.preferIPv4Addresses=true
openjdk version "1.8.0_102"
OpenJDK Runtime Environment (build 1.8.0_102-8u102-b14.1-1-b14)
OpenJDK 64-Bit Server VM (build 25.102-b14, mixed mode)

$ lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux testing (stretch)
Release:	testing
Codename:	stretch
```

**Comment by 00-Evan** [comment]
I've actually just released 0.4.1a to desktop, do you get the same crashes on the latest version?

**Comment by rohieb** [comment]
No, everything is fine, thanks! :)
